### PR TITLE
🌐 Add Chinese translation for `docs/zh/docs/deployment/versions.md`

### PR DESCRIPTION
**Pull request by xzmeng**

### CODE REVIEW

**Comment by tiangolo** [comment]
📝 Docs preview for commit 754d4ff20b27678b80449936f9aa8a2a28933476 at: https://c29b0395.fastapitiangolo.pages.dev

File: docs/zh/docs/deployment/versions.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
+# 关于 FastAPI 版本
+
+**FastAPI** 已在许多应用程序和系统的生产环境中使用。 并且测试覆盖率保持在100%。 但其开发进度仍在快速推进。
+
+经常添加新功能，定期修复错误，并且代码仍在持续改进。
+
+这就是为什么当前版本仍然是“0.x.x”，这反映出每个版本都可能有Breaking changes。 这遵循<a href="https://semver.org/" class="external-link" target="_blank">语义版本控制</a>的约定。
```
Comment (shamosishen) [inline docs/zh/docs/deployment/versions.md:7]:
```suggestion
这就是为什么当前版本仍然是`0.x.x`，这反映出每个版本都可能有Breaking changes。 这遵循<a href="https://semver.org/" class="external-link" target="_blank">语义版本控制</a>的约定。
```

File: docs/zh/docs/deployment/versions.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+你现在就可以使用 **FastAPI** 创建生产环境应用程序（你可能已经这样做了一段时间），你只需确保使用的版本可以与其余代码正确配合即可。
+
+## 固定你的 `fastapi` 版本
+
+你应该做的第一件事是将你正在使用的 **FastAPI** 版本“固定”到你知道适用于你的应用程序的特定最新版本。
+
+例如，假设你在应用程序中使用版本`0.45.0`。
+
+如果你使用“requirements.txt”文件，你可以使用以下命令指定版本：
```
Comment (shamosishen) [inline docs/zh/docs/deployment/versions.md:17]:
```suggestion
如果你使用`requirements.txt`文件，你可以使用以下命令指定版本：
```

File: docs/zh/docs/deployment/versions.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+## 升级FastAPI版本
+
+你应该为你的应用程序添加测试。
+
+使用 **FastAPI** 编写测试非常简单（感谢 Starlette），请参考文档：[测试](../tutorial/testing.md){.internal-link target=_blank}
+
+添加测试后，你可以将 **FastAPI** 版本升级到更新版本，并通过运行测试来确保所有代码都能正常工作。
+
+如果一切正常，或者在进行必要的更改之后，并且所有测试都通过了，那么你可以将“fastapi”固定到新的版本。
```
Comment (shamosishen) [inline docs/zh/docs/deployment/versions.md:67]:
```suggestion
如果一切正常，或者在进行必要的更改之后，并且所有测试都通过了，那么你可以将`fastapi`固定到新的版本。
```

File: docs/zh/docs/deployment/versions.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+
+你不应该固定`starlette`的版本。
+
+不同版本的 **FastAPI** 将使用特定的较新版本的 Starlette。
+
+因此，**FastAPI** 自己可以使用正确的 Starlette 版本。
+
+## 关于 Pydantic
+
+Pydantic 包含针对 **FastAPI** 的测试及其自己的测试，因此 Pydantic 的新版本（“1.0.0”以上）始终与 FastAPI 兼容。
```
Comment (shamosishen) [inline docs/zh/docs/deployment/versions.md:79]:
```suggestion
Pydantic 包含针对 **FastAPI** 的测试及其自己的测试，因此 Pydantic 的新版本（`1.0.0`以上）始终与 FastAPI 兼容。
```

**Review by shamosishen — CHANGES_REQUESTED** [review]
修改一些与原文不一致的地方

**Comment by tiangolo** [comment]
📝 Docs preview for commit 029665811697a3c211dd71f0b0ad7a21007438c1 at: https://70291246.fastapitiangolo.pages.dev

**Review by shamosishen — APPROVED** [review]

**Review by wdh99 — APPROVED** [review]

**Comment by tiangolo** [comment]
Great, thanks @xzmeng! 🙇 

And thanks for the reviews @shamosishen and @wdh99 🍰

**Pull request merged**